Tests/TuistKitTests/Services/ShareServiceTests.swift

```diff
@@ -22,6 +22,8 @@ final class ShareServiceTests: TuistUnitTestCase {
     private var userInputReader: MockUserInputReading!
     private var defaultConfigurationFetcher: MockDefaultConfigurationFetching!
     private var appBundleLoader: MockAppBundleLoading!
+    private var fileUnarchiver: MockFileUnarchiving!
+    private let shareURL: URL = .test()
```

Comment:
Don't forget to set these to `nil` in `tearDown` for the garbage collector to clean up the memory.

Reply:
I don't think it's necessary for constants.